src/redux/user/userSlice.ts

```diff
@@ -14,6 +14,16 @@ const initialState: IUser = {
   accessToken: '',
   refreshToken: '',
   isAccountActive: true,
+  photoUrl: '',
+  phoneNumber: '',
+  addressLine1: '',
+  addressLine2: '',
+  country: '',
+  state: '',
+  city: '',
+  clothesSize: '',
+  jeansSize: '',
+  shoesSize: '',
 };
```

Comment:
why we cant make initial state just empty object {}, without setup all fields, I dont like current approach

Reply:
fixed

Reply:
in some reducers we address to a state props like: 
setEmail(state, action) {
      state.email = action.payload;
    },
    
and without initial state I have errors with it